src/analysis/retail/priest/discipline/modules/spells/BlazeOfLight/BlazeOfLightSourceDonut.tsx

```diff
@@ -33,7 +36,7 @@ function generateHealingItems(
     return {
       color: `${COLORS[ability.guid]}`,
       spellId: ability.guid,
-      label: ability.name,
+      label: getAbilityNameReporting(ability),
```

Comment:
this could be `getSpell(ability.guid).name`

Reply:
Probably a better solution. Wasn't aware of this being valid.
There are duplicated spells with different ids but same name on logs that I wanted to differentiate on the report to make it less confusing to read, in this case I differentiated it inside of the spell list so this would actually work fine.